cryptocurrency-subdomain/src/main/kotlin/com/ajax/cryptocurrency/application/ports/repository/CryptocurrencyRepository.kt

```diff
@@ -0,0 +1,20 @@
+package com.ajax.cryptocurrency.application.ports.repository
+
+import com.ajax.cryptocurrency.domain.CryptocurrencyDomain
+import org.springframework.data.domain.Pageable
+import org.springframework.stereotype.Repository
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+
+@Repository
```

Comment:
why do you need an annotation on an interface? what if exact implementation won't be a repository and will read from in-memory cache?

Reply:
simple inattention, reworked
thx for the correction